pallets/migration/src/lib.rs

```diff
@@ -175,6 +192,11 @@ pub mod pallet {
 		) -> DispatchResultWithPostInfo {
 			ensure_root(origin)?;
 
+			ensure!(
+				<Completed<T>>::get() == false,
```

Comment:
`nit`: It is more idiomatic to use `! expr` than `expr == false` 
```suggestion
				!<Completed<T>>::get(),
```

Reply:
I likem will change. Although with the new enum.